doc/changelog.rst

```diff
@@ -11,56 +11,125 @@ Jump to :ref:`SmartRedis Changelog <sr_changelog>`
 SmartSim
 ========
 
-
-Development branch
-------------------
+0.6.1
+-----
 
 To be released at some future point in time
 
 Description
 
-- Updated tests to address Torch warning
-- Updated GitHub actions to latest versions in CI/CD
-- Dropped Cobalt support
-- Override the sphinx-tabs extension background color
-- Updated SmartSim's machine learning backends
-- Added ONNX support for Python 3.10
-- Added support for Python 3.11
-- Added support for SmartSim with Torch on Apple Silicon
-- Updated install instructions for Apple Silicon
+- Fix line-ending related errors on MacOS (ARM64)
+- Update instructions for Apple Silicon
+- Duplicate DBModel/Script prevention
+- Update license to include 2024
+- Patch changes for release prep
```

Comment:
I think the only thing is this PR that someone would reading the change log on would be that the TM is now on by default. Can we update this description to
```suggestion
- Telemetry monitor is now active by default
```
or something similar?